applications/StructuralMechanicsApplication/custom_python/structural_mechanics_python_application.cpp

```diff
@@ -204,6 +204,9 @@ PYBIND11_MODULE(KratosStructuralMechanicsApplication,m)
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m,INELASTIC_FLAG)
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m,INFINITY_YIELD_STRESS)
 
+    // Error estimate
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, ERROR_ESTIMATE)
```

Comment:
isn't there a ERROR_RATIO variable defined in the core ?

Reply:
I don't know